testdata/scripts/generate_unexisting_import.txt

```diff
@@ -0,0 +1,14 @@
+# make sure go modules is turned off, and GOPATH is set to a nonexistent directory
```

Comment:
Just noticed "unexisting" in this file name - the correct term would be "nonexistent"

Reply:
Or even better, "missing" :)

Reply:
> Just noticed "unexisting" in this file name - the correct term would be "nonexistent"
 
changed all to nonexistent.